documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -146,14 +150,76 @@ Example CPU images:
 | container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol7-211124 | JDK 8u311, Oracle Linux 7u9, and Oracle WebLogic Server 12.2.1.4 generic distribution October 2021 CPU |
 | container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol7 | Represents latest JDK 8, latest Oracle Linux 7, and GA Oracle WebLogic Server 12.2.1.4 generic distribution CPU |
 
+You may have noticed that the image tags may include keywords like `generic`, `slim`, etc.
+This reflects the type of WebLogic install in the image. There are multiple types,
```

Comment:
WebLogic install -> WebLogic installation

Reply:
now "type of WebLogic distribution installed in the image's Oracle Home." (text now moved to images doc)